src/index.ts

```diff
@@ -84,11 +135,43 @@ if (process.env.NODE_ENV === 'stub') {
   app.get('/settings', protect, settingsHandler)
   app.get('/verify', verifyHandler)
   app.get('/recovery', recoveryHandler)
+
+  if (Boolean(config.hydra.admin)) {
+    app.get('/auth/hydra/login', hydraLogin)
+    app.get(
+      '/auth/hydra/consent',
+      protect,
+      csrfProtection,
+      hydraGetConsent,
+      errorHandler
+    )
+    app.post(
+      '/auth/hydra/consent',
+      protect,
+      bodyParser.urlencoded({ extended: true }),
+      csrfProtection,
+      hydraPostConsent
+    )
+  }
 }
 
 app.get('/health', (_: Request, res: Response) => res.send('ok'))
 app.get('/debug', debug)
 
+if (config.securityMode === SECURITY_MODE_STANDALONE) {
```

Comment:
Do we need this? We removed it as it no longer had a purpose!

Reply:
Based on how the integration was originally implemented, this proxying is required.  The browser does not speak to Kratos directly, it always goes through the selfservice-ui.

Reply:
Ok, thank you for the clarification!